Challenges/Day_1/solution_day1_script.sh

```
@@ -1,31 +1,41 @@
 
-# First line of the script is the shebang which tells the system how to execute
 
-# Task  2: Echo
-echo "Scripting is fun with @TWS"
+#Task1:Comment
+#by this hashtag we can give oneline comment and exclude any particular line of code by this.
 
-# Task  3: Variables
-variable1="Hello"
-variable2="Bash"
+<<comment
```

Shellcheck

[warning] This redirection doesn't have a command. Move to its command (or use 'true' as no-op).

(SC2188)